Portfolio.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <a id="logo" href="">My Logo</a>
+        <nav>
+            <img class="tools" src="../Axel-Soler-Portfolio/images/Union.png" alt="hamburguer">
```

Comment:
- [ ][OPTIONAL] At the moment, this hamburger icon is not being displayed because the relative path is not correct. Kindly use `./images/Union.png`